tests/lib/rules/no-unused-prop-types.js

```diff
@@ -2440,6 +2440,17 @@ ruleTester.run('no-unused-prop-types', rule, {
       }
       `,
       parser: 'babel-eslint'
+    },
+    {
+      code: `
+        class MyComponent extends React.Component<Props> {
+          render() {
+            return <div>{ this.props.other }</div>
+          }
+        }
+        MyComponent.propTypes = { other: () => {} };
+      `,
+      parser: 'babel-eslint'
```

Comment:
this shouldn't need babel-eslint; can you either add a duplicate test case (one using the default parser, and one using babel-eslint), or, remove the parser line and only test using the default parser?

Reply:
You're right, I'll take out the parser line and leave the test cases using the default parser